<!-- Launch file to detect charuco markers -->

<launch>
    <group ns="saf">

    <node name="qr_code_position" pkg="irob_vision_support"
            type="qr_code_position.py" output="screen" >

               <!--  <remap from="image_input" to="mono/cam/image_raw" />
                <remap from="image_markers" to="vision/image_markers" />
                <remap from="markers" to="vision/markers" />
                <remap from="camera_info" to="mono/cam/camera_info" />
                <remap from="transform" to="vision/transform" /> -->

                <remap from="image_input" to="stereo/left/image_color" />

                <remap from="markers" to="qrkam" />

    </node>
    </group>
</launch>

